arbnode/delayed_sequencer.go

```diff
@@ -52,14 +52,14 @@ func DelayedSequencerConfigAddOptions(prefix string, f *flag.FlagSet) {
 var DefaultDelayedSequencerConfig = DelayedSequencerConfig{
 	Enable:              false,
 	FinalizeDistance:    20,
-	RequireFullFinality: true,
+	RequireFullFinality: false,
```

Comment:
I think this should be a single (string/enum) flag rather than having two flags and deciding what to do based on the combination of these two booleans.

Not gonna block pr on this though.